data/mods/sssb/abilities.js

```diff
@@ -275,6 +275,56 @@ let BattleAbilities = {
 				this.useMove("healbell", pokemon);
 			}
 		},
+	},
+	
+	// Revival xFloatz
+	"xfz": {
+		id: "xfz",
+		name: "XFZ",
+		desc: "Flash Fire and Prankster.",
```

Comment:
I'd prefer descriptions actually reflect what they are not just say the combination of abilities